VideoLocker/res/values/strings.xml

```diff
@@ -477,6 +477,14 @@
     <!-- In Sign up screen -->
     <string name="or_sign_up_with_email">or sign up with email</string>
 
+    <!-- New Logistration Launch screen text -->
+    <!-- Button leading user to create account screen -->
+    <string name="launch_register">Register</string>
+    <!-- Button leading user to Find Courses screen from the launch screen -->
+    <string name="launch_discover_courses">@string/find_courses_title</string>
```

Comment:
@bguertin good point. Marketing wants the button to be call "Discover Courses" but I agree about keeping consistency. 

@BenjiLee can we ask Janaky if it makes sense to change the edx.org btn to "Discover Courses" to keep them consistent?